apps/testnorge-sigrun/sigrun-core/src/test/java/no/nav/registre/sigrun/provider/rs/IdentControllerIntegrationTest.java

```diff
@@ -0,0 +1,27 @@
+package no.nav.registre.sigrun.provider.rs;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.github.tomakehurst.wiremock.client.WireMock;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.contract.wiremock.AutoConfigureWireMock;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@AutoConfigureWireMock(port = 0)
+@AutoConfigureMockMvc
+@DirtiesContext
+@TestPropertySource(locations = "classpath:application-test.properties")
+class IdentControllerIntegrationTest {
```

Comment:
Se kommentarer for organisasjonControllerIntegrationTest.

Kan også fjerne SyntetiseringControllerIntegrationTest sånn at det bare er en tom test i testnorge-sigrun.